src/applications/vaos/actions/newAppointment.js

```diff
@@ -18,3 +24,42 @@ export function updateFormData(page, uiSchema, data) {
     data,
   };
 }
+
+export function routeToPage(flow, router, current, action) {
+  return async (dispatch, getState) => {
+    dispatch({
+      type: FORM_PAGE_CHANGE_STARTED,
+    });
+
+    const nextAction = flow[current][action];
+    let nextState;
+
+    if (typeof nextAction === 'string') {
+      nextState = flow[nextAction];
+    } else {
+      const nextStateKey = await nextAction(getState(), dispatch);
+      nextState = flow[nextStateKey];
+    }
+
+    if (nextState?.url) {
+      router.push(nextState.url);
+      dispatch({
+        type: FORM_PAGE_CHANGE_COMPLETED,
+      });
+    } else if (nextState) {
+      throw new Error(`Tried to route to a page without a url: ${nextState}`);
+    } else {
+      throw new Error('Tried to route to page that does not exist');
+    }
+
+    return nextState;
```

Comment:
where does this go?

Reply:
Nowhere, I suppose I could just remove it.

Reply:
I figured it was for testing but didn't know.